specification/monitor/resource-manager/microsoft.insights/stable/2018-04-16/scheduledQueryRule_API.json

```diff
@@ -0,0 +1,628 @@
+            "$ref": "#/parameters/RuleNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/LogSearchRuleResource"
+            },
+            "description": "The parameters of the rule to create or update."
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "200": {
+            "description": "Successful request to update an Log Search rule",
+            "schema": {
+              "$ref": "#/definitions/LogSearchRuleResource"
+            }
+          },
+          "201": {
+            "description": "Created alert rule",
+            "schema": {
+              "$ref": "#/definitions/LogSearchRuleResource"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Create or Update rule": { "$ref": "./examples/createOrUpdateScheduledQueryRules.json" }
+        }
+      },
+      "get": {
+        "description": "Gets an Log Search rule",
+        "tags": [
+          "scheduledQueryRules"
+        ],
+        "operationId": "ScheduledQueryRules_Get",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/RuleNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "200": {
+            "description": "Successful request to get a Log Search rule",
+            "schema": {
+              "$ref": "#/definitions/LogSearchRuleResource"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get rule": { "$ref": "./examples/getScheduledQueryRules.json" }
+        }
+      },
+      "delete": {
+        "description": "Deletes a Log Search rule",
+        "tags": [
+          "scheduledQueryRules"
+        ],
+        "operationId": "ScheduledQueryRules_Delete",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/RuleNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
```

Comment:
You should have a `default` response modeled here -- if the service fails for any reason, the client needs to be able to get the error response.